### ISSUE TITLE
Background in Blockly Games Bird

### ISSUE THREAD

**Issue by annasindarsin** [google/blockly-games#65]
Hello.
I want to give background to my Blockly Games Bird. But I can't seem to find where to add backgrounds in bird.js, on the Bird.drawMap() function (which I believe is used to render sprites (bird, worm, nest, walls) on the game). Please help me solve this problem. Thank you for your attention.

**Comment by Vancer** [comment]
@annasindarsin  Could you please note down how did you fix it ?

**Comment by annasindarsin** [comment]
My way to fix is kind of brute force, but I hope it helps anyway.
I declare a new variable for background on bird.drawMap() function and set the id attribute to 'background'.
Then, I used a series of 'if' block to show different background images on each level.
finally, I set the height and width attribute to 400 and appendChild(bg) to var svg. 
Here's the code snippet:
`var bg = document.createElementNS(Blockly.SVG_NS, 'image');
bg.setAttribute('id', 'background');
  if(BlocklyGames.LEVEL == 1){
  	bg.setAttributeNS('http://www.w3.org/1999/xlink', 'xlink:href',
    '<<path to bg picture>>');
  } else if(BlocklyGames.LEVEL == 2){
  //same as previous
.
.
.
}
  bg.setAttribute('height', 400);
  bg.setAttribute('width', 400);
  svg.appendChild(bg);`

hope this helps.
